regression/cbmc/Makefile

```diff
@@ -1,13 +1,12 @@
 default: tests.log
 
-test:
-	@../test.pl -p -c ../../../src/cbmc/cbmc -X smt-backend
+test: tests.log
```

Comment:
This is tempting, but I think it's wrong: we do not have proper dependency tracking here (tests should be re-run whenever either the binary or the tests themselves change).

Reply:
Ok, I'll simply make "test" the default target.